docs/node-operators/get-started/setup-validator.md

```diff
@@ -232,6 +232,29 @@ When a Vega validator node is watching for Ethereum events it will call the `eth
 ```
 
 
+:::warning
+You must select a big enough value for the `MaxEthereumBlocks` because Arbitrum produces about five blocks per second. If the range is too small, your node will not keep up with the rest of the network.
+:::
+
+:::note Find blocks spam for your RPC provider
+You must check how big the span of the blocks you can use with your provider. Otherwise, your validator won't be able to validate events from the Arbitrum network.
+
+The default value(10000) is usually more than enough. If your provider limits it, you should use the maximum allowed range. 
```

Comment:
haven't we set it to 500 in our testnet node? Should we mention that some does limit to 500 and which they are. Debugging this on mainnet node will be a real pain, so maybe we want to mention or something.